lib/render/vp2RenderDelegate/render_delegate.cpp

```diff
@@ -344,6 +335,14 @@ HdVP2RenderDelegate::HdVP2RenderDelegate(ProxyRenderDelegate& drawScene) {
     std::lock_guard<std::mutex> guard(_renderDelegateMutex);
     if (_renderDelegateCounter.fetch_add(1) == 0) {
         _resourceRegistry.reset(new HdResourceRegistry());
+
+        // HdVP2BBoxGeom can only be instantiated during the lifetime of VP2
+        // renderer from main thread. HdVP2RenderDelegate is created from main
+        // thread currently, if we need to make its creation parallel in the
+        // future we should move this code out.
+        if (TF_VERIFY(sSharedBBoxGeom == nullptr)) {
+            sSharedBBoxGeom = new HdVP2BBoxGeom();
+        }
```

Comment:
I couldn't repro the crash in AL unit test crash in Maya prompt mode. The callstack appears to indicate HdVP2BBoxGeom was instantiated before VP2 initialization. Thus I make this change to make sure it is instantiated after VP2 initialization.

Reply:
AND @cfmoore007 helped to validate the fix before I pushing the commit.

Reply:
Makes sense!